packages/wonder-blocks-dropdown/components/action-menu-opener-core.js

```diff
@@ -0,0 +1,183 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import PropTypes from "prop-types";
+
+import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+import Color, {SemanticColor, mix} from "@khanacademy/wonder-blocks-color";
+import {addStyle} from "@khanacademy/wonder-blocks-core";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+import type {ClickableHandlers} from "@khanacademy/wonder-blocks-core";
+import type {SharedProps} from "./action-menu-opener.js";
+
+type Props = {|
+    ...SharedProps,
+    ...ClickableHandlers,
+    hovered: boolean,
+    focused: boolean,
+    pressed: boolean,
+|};
+
+const StyledButton = addStyle("button");
+
+export default class ActionMenuOpenerCore extends React.Component<Props> {
```

Comment:
It feels like a lot of this is duplicated from Button - could you add a comment explaining what's the same and what's different?

Reply:
I'll add some comments.

Reply:
The main reason was that the icon appears on the right side and the icon we use is a different size from what the Button component would otherwise use.